src/java.desktop/share/classes/javax/swing/text/html/CSS.java

```diff
@@ -2823,6 +2907,20 @@ boolean isVerticalPositionRelativeToFontSize() {
         float getVerticalPosition() {
             return verticalPosition;
         }
+
+        @Override
+        public int hashCode() {
+            int hashCode = Float.hashCode(horizontalPosition);
+            hashCode |= Float.hashCode(verticalPosition);
+            return hashCode;
+        }
+
+        @Override
+        public boolean equals(Object val) {
+            return val instanceof CSS.BackgroundPosition bp
```

Comment:
why isn't "relative" part of equals and hashCode()?

Reply:
I have added "relative" to equals/hashCode